src/cryptoadvance/specter/api/rest/api.py

```diff
@@ -16,6 +16,7 @@
 from ...util.fee_estimation import get_fees
 
 from .. import auth
+from .jwt import TokenResource
```

Comment:
Please apply the naming conventions. Check the other resources for examples. In that case you need to rename the file.